ducktape/tests/reporter.py

```diff
@@ -284,12 +299,16 @@ def format_report(self):
             'num_passes': self.results.num_passed,
             'num_failures': self.results.num_failed,
             'num_ignored': self.results.num_ignored,
+            'num_opassed': self.results.num_opassed,
+            'num_ofailed': self.results.num_ofailed,
             'run_time': format_time(self.results.run_time_seconds),
             'session': self.results.session_context.session_id,
             'passed_tests': passed_result_string,
             'failed_tests': failed_result_string,
             'ignored_tests': ignored_result_string,
-            'test_status_names': ",".join(["\'%s\'" % str(status) for status in [PASS, FAIL, IGNORE]])
+            'ofailed_tests': ofailed_result_string,
+            'opassed_tests': opassed_result_string,
+            'test_status_names': ",".join(["\'%s\'" % str(status) for status in [PASS, FAIL, IGNORE, OPASS, OFAIL]])
```

Comment:
nit: brackets unnecessary
```suggestion
            'test_status_names': ",".join("\'%s\'" % str(status) for status in [PASS, FAIL, IGNORE, OPASS, OFAIL])
```